application/controllers/UserManagementController.php

```diff
@@ -67,10 +67,14 @@ public function actionIndex()
             false
         );
 
+        $oEvent = new PluginEvent('beforeAdminUserActivation');
+        $oEvent->set('request', App()->request);
+        App()->getPluginManager()->dispatchEvent($oEvent);
+        $isUserLimitReached = $oEvent->get('disable', null);
```

Comment:
User limit checks must be moved to the LimeSurvey Professional plugin on LS Cloud repo.